basketball_reference_web_scraper/html.py

```diff
@@ -870,6 +870,17 @@ def game_url_paths(self):
         game_links = self.html.xpath(self.game_url_paths_query)
         return [game_link.attrib['href'] for game_link in game_links]
 
+class TeamRoster:
+    def __init__(self, html):
+        self.html = html
+
+    @property
+    def roster_query(self):
+        return '//table[@id="roster"]//td[@data-stat="player"]'
+    @property
+    def team_roster(self):
```

Comment:
Let's create `Row` classes to represent the underlying row content (to keep consistent with similar patterns in this file).